public/locales/en.json

```diff
@@ -169,6 +169,7 @@
     "routingHelper": "Routing through these tokens resulted in the best price for your trade.",
     "confirmSwap": "Confirm Swap",
     "swapAnyway": "Swap Anyway",
+    "confirm": "confirm",
```

Comment:
we need to also include `searchModal.remove` `settings.confirm` variables for English